lib/worker.js

```diff
@@ -282,11 +282,18 @@ class AssetComputeWorker {
         try {
             if (await this.shouldPostProcess(rendition)) {
 
+                if (rendition.instructions.watermark && rendition.instructions.watermark.watermarkContent !== undefined && rendition.instructions.watermark.watermarkContent !== null) {
+                    rendition.instructions.watermark = await getWatermark(
+                        rendition.instructions.watermark,
+                        this.directories.in,
+                        this.options.disableSourceDownload
+                    );
+                }
```

Comment:
See https://github.com/adobe/asset-compute-sdk/pull/57#pullrequestreview-484668073 - such watermarking specific logic should be encapsulated in the post processing code only, i.e. `image.js` and co.